engine/api/application/gorp_model.go

```diff
@@ -14,13 +14,24 @@ import (
 
 type dbApplication sdk.Application
 type dbApplicationVariableAudit sdk.ApplicationVariableAudit
-type dbApplicationKey sdk.ApplicationKey
+type dbApplicationKey struct {
+	gorpmapping.SignedEntity
+	sdk.ApplicationKey
+}
+
+func (e dbApplicationKey) Canonical() gorpmapping.CanonicalForms {
+	var _ = []interface{}{e.ApplicationID, e.ID, e.Name}
+	return gorpmapping.CanonicalForms{
+		"{{print .ApplicationID}} {{print .ID}} {{.Name}}",
```

Comment:
Weed didn't add spaces for other entities

Reply:
weed ?

Reply:
🚬🔥

Reply:
done